src/tonic/api/points_api.rs

```diff
@@ -53,7 +53,13 @@ impl Points for PointsService {
 
     async fn get(&self, request: Request<GetPoints>) -> Result<Response<GetResponse>, Status> {
         validate(request.get_ref())?;
-        get(self.toc.as_ref(), request.into_inner(), None).await
+
+        let mut request = request.into_inner();
+        request
+            .read_consistency
+            .get_or_insert(default_read_consistency()); // *Have* to be `Some`!
```

Comment:
I don't think there's any reason to use `get_or_insert_with` instead of `get_or_insert`.

The main point of `get_or_insert_with` is if it's "expensive" to construct the new value. And so you provide a closure, that can be called _conditionally, if/when it's actually needed_. But `ReadConsistency` is just a `Copy`-able enum. It's cheap to make (and move), so it makes no sense to use `get_or_insert_with`.

Reply:
In this case, fair point! Thanks for the explanation. Indeed I suggested this for being potentially expensive.